code/_core/obj/structure/interactive/nature/snow.dm

```diff
@@ -113,6 +115,8 @@ obj/structure/scenery/snow_bush/New()
 
 /obj/structure/interactive/tree/jungle_small/New(var/desired_loc)
 	icon_state = "tree[rand(1,6)]"
+	pixel_x = rand(-33,-30)
+	pixel_y = rand(0,2)
```

Comment:
this could probably be put on the /obj/structure/interactive/tree interactive structure instead.